wp-content/themes/pub/wporg-learn-2020/template-parts/component-video-grid-item.php

```diff
@@ -10,10 +10,7 @@
 ?>
 
 <li class="col-4 video-grid_item">
-	<?php if( has_post_thumbnail() ) : ?>
-		<?php echo the_post_thumbnail( 'medium' ); ?>
-	<?php else : ?>
-		<div class="video-grid_item--no-image"></div>
+		<?php echo wporg_get_post_thumbnail( $post, 'medium' );; ?>
 	<? endif; ?>
```

Comment:
Looks like the `endif;` should've gone here too? and while i'm here, double `;;`

Reply:
Ugh, yeah, thanks - copy-paste errors.